pino.js

```diff
@@ -211,11 +214,13 @@ pino.destination = (dest = process.stdout.fd) => {
   }
 }
 
-pino.final = final
-pino.levels = mappings()
-pino.stdSerializers = serializers
-pino.stdTimeFunctions = Object.assign({}, time)
-pino.symbols = symbols
-pino.version = version
+module.exports.final = final
+module.exports.levels = mappings()
+module.exports.stdSerializers = serializers
+module.exports.stdTimeFunctions = Object.assign({}, time)
+module.exports.symbols = symbols
+module.exports.version = version
```

Comment:
I don't understand how this is any different than the prior code.

Reply:
Node.js now makes static analysis of your exports and if it finds `module.exports` it adds them as named export in the ESM context. The result in JS is the same as before, but writing it like this helps module interop!

Reply:
![image](https://user-images.githubusercontent.com/321201/101908458-2065cc00-3b8a-11eb-8ef7-a298c897adee.png)

Reply:
ahahahah <3